django_x509/tests/base/test_admin.py

```diff
@@ -90,24 +87,24 @@ def has_perm(self, perm):
                  'modified']
 
 
-class ModelAdminTests(TestCase):
+class AbstractModelAdminTests(object):
 
     def setUp(self):
-        self.ca = Ca.objects.create()
-        self.cert = Cert.objects.create(ca_id=self.ca.pk)
+        self.ca = self.ca_model.objects.create()
```

Comment:
if we use swappable models we don't need to do this anymore

Reply:
Okay, it comes with consequences though: please check:
```
def _create_cert(self, cert_model=None, **kwargs):
        if not cert_model:
            cert_model = Cert
```
in tests/base.py :smile: